src/Core.Tests/GenericContainerWithVolumneTests.cs

```diff
@@ -28,19 +31,35 @@ public void PrepareResource_NoError()
             //Assert
             action.Should().NotThrow();
         }
+
+        [Fact]
+        public async Task PrepareResource_VolumeMapped()
+        {
+            //Arrange
+            Uri externalUri = _resource.GetContainerUri();
+            using var client = new HttpClient();
+
+            //Act
+            using HttpResponseMessage response = await client.GetAsync(externalUri);
+            response.EnsureSuccessStatusCode();
+            var content = await response.Content.ReadAsStringAsync();
+
+            //Assert
+            content.Should().Contain("Hello Squadron unit test!");
```

Comment:
You could actually make a snapshot here with Snapshooter but it's only a suggestion. It's ok so.